pkg/builder/openapi.go

```diff
@@ -100,6 +102,7 @@ func newOpenAPI(config *common.Config) openAPI {
 				Responses:   config.ResponseDefinitions,
 				Paths:       &spec.Paths{Paths: map[string]spec.PathItem{}},
 				Info:        config.Info,
+				Parameters:  config.SharedParameters,
```

Comment:
I know this is a WIP PR, but the parameters shouldn't be defined in the config imo. We should be able to calculate the set of parameters purely through the building process.